app/views/teams/leaderboard.html.erb

```diff
@@ -19,7 +20,11 @@
     <% end %>
   </tbody>
 </table>
-
 <br>
-
-<%= link_to 'New Team', new_team_path %> | <%= link_to 'New bill', new_bill_path %>
\ No newline at end of file
+<%= link_to 'New Team', new_team_path, class: "button" %> <%= link_to 'New bill', new_bill_path, class: "button" %>
+</div>
+<div class="right">
+<h2>Money Saving Tip of the Day</h2>
+Rock that beach body in your living room! Keep your air conditioner to 78 degrees or higher in the summer and use a fan instead.
+<p class="button">I did this!</div>
```

Comment:
This is a not at all functional button but might have encouragement or switch to a different tip later.